<launch>
	<node pkg="stereo_click" type="click_window.py" name="click_window_right">
		<param name="cam" value="/wide_stereo/left" />
		<param name="output" value="clicks_wide_stereo_left" />
	</node>

	<node pkg="stereo_click" type="optical_converter.py" name="optical_converter">
		<param name="input" value="clicks_wide_stereo_left" />
		<param name="output" value="stereo_points_3d" />
		<param name="output_frame" value="odom_combined" />
	</node>
</launch>
